addons/l10n_it/data/account.tax.group.csv

```diff
@@ -1,26 +1,23 @@
 id,name
-tax_group_iva_2,IVA 2%
 tax_group_iva_4,IVA 4%
 tax_group_iva_10,IVA 10%
-tax_group_iva_12,IVA 12%
-tax_group_iva_21,IVA 21%
-tax_group_iva_20,IVA 20%
 tax_group_iva_22,IVA 22%
 tax_group_iva_4_de_50,IVA 4% Detraibile 50%
 tax_group_iva_10_de_50,IVA 10% Detraibile 50%
-tax_group_iva_20_de_10,IVA 20% Detraibile 10%
-tax_group_iva_20_de_15,IVA 20% Detraibile 15%
-tax_group_iva_20_de_40,IVA 20% Detraibile 40%
-tax_group_iva_20_de_50,IVA 20% Detraibile 50%
-tax_group_iva_21_inde,IVA 21% Indetraibile
-tax_group_iva_21_de_10,IVA 21% Detraibile 10%
-tax_group_iva_21_de_15,IVA 21% Detraibile 15%
-tax_group_iva_21_de_40,IVA 21% Detraibile 40%
-tax_group_iva_21_de_50,IVA 21% Detraibile 50%
 tax_group_iva_22_inde,IVA 22% Indetraibile
 tax_group_iva_22_de_10,IVA 22% Detraibile 10%
 tax_group_iva_22_de_15,IVA 22% Detraibile 15%
 tax_group_iva_22_de_40,IVA 22% Detraibile 40%
 tax_group_iva_22_de_50,IVA 22% Detraibile 50%
 tax_group_imp_esc_art_15,Imponibile Escluso Art.15
-tax_group_fuori,Fuori Campo IVA
+tax_group_fuori,Fuori campo IVA
+tax_group_imp_ese_art_8,Non imponibile Art.8
+tax_group_imp_ese_art_2,Fuori campo IVA Art.2
+tax_group_imp_ese_art_9,Non imponibile Art.9
+tax_group_imp_ese_art_41,Non imponibile Art.41
+tax_group_imp_ese_art_71,Non imponibile Art.71
+tax_group_imp_ese_art_74,Imponibile escluso Art.74
+tax_group_imp_ese_art_7,Fuori campo IVA Art.7
+tax_group_imp_ese_art_10,Imponibile Esente Art.10
+tax_group_imp_ese_art_58,Non imponibile Art.58
+tax_group_imp_ese_art_196,Non imponibile Art.196
```

Comment:
As for wiki translation guidelines
Imponibile Esente → Imponibile esente
Art. → art.